guides/common/modules/con_iss-import-export-cheat-sheet.adoc

```diff
@@ -9,6 +9,10 @@
 |Fully Export a Content View version | `hammer content-export complete version --content-view="_My_Content_View_" --version=1.0 --organization="_My_Organization_"`
 |Incrementally Export a Content View version (assuming you have already exported something)| `hammer content-export incremental version --content-view="_My_Content_View_" --version=2.0 --organization="_My_Organization_"`
 
+|Fully Export a Repository| `hammer content-export complete library --product="_My_Product_" --name="_My_Repository" --organization="_My_Organization_"`
+
+|Incrementally Export a Repository (assuming you have already exported something)|`hammer content-export incremental --product="_My_Product_" --name="_My_Repository" --organization="_My_Organization_"`
```

Comment:
What to export is missing here.
`hammer content-export incremental repository ...`